akka-projection-core/src/main/resources/reference.conf

```diff
@@ -13,4 +13,20 @@ akka.projection {
     save-offset-after-envelopes = 100
     save-offset-after-duration = 500 ms
   }
+
+  recovery-strategy {
+    # fail - Fail the projection if the handler throws an exception
+    # skip - Skip to the next envelope
+    # retry-and-fail - Retry the handler with the same envelope `retries` before failure
+    # retry-and-skip - Retry the handler with the same evvelope `retries` before moving to the next record
+    strategy = fail
+
+    # The Number of times to retry handler function
+    # Only applicable to `retry-and-fail` and `retry-and-skip` recovery strategies
+    retries = 1
+
+    # Delay between retry attempts
+    # Only applicable to `retry-and-fail` and `retry-and-skip` recovery strategies
+    retry-delay = 0 s
```

Comment:
we should have better defaults for `retries` and `retry-delay` so that it would be valid to only configure `strategy = retry-and-fail`.

```
retries = 5
retry-delay = 1 s
```